platform_versioned_docs/version-23.4.0/enterprise/prerequisites/gcp.mdx

```diff
@@ -57,72 +47,131 @@ A [Google Kubernetes Engine (GKE)](https://cloud.google.com/kubernetes-engine) c
 Seqera doesn't currently support GKE Autopilot due to a privilege issue with the Redis deployment. However, you can achieve most of the same behavior with a Standard cluster by enabling autoscaling and node auto-provisioning.
 :::
 
-### Seqera container images
+## GCP setup
 
-Seqera Platform Enterprise is distributed as a collection of Docker containers available through the Seqera
-container registry [`cr.seqera.io`](https://cr.seqera.io). Contact [support](https://support.seqera.io) to get your container access credentials. Once you've received your credentials, retrieve the Seqera container images:
+This section provides step-by-step instructions for some commonly used GCP services for Seqera deployment. See the [GCP documentation](https://cloud.google.com/docs) for up-to-date instructions and contact [GCP support](https://cloud.google.com/support-hub) if you have any issues with provisioning GCP resources.
 
-1. Retrieve the **username** and **password** you received from Seqera support.
+### Google CloudSQL
 
-2. Run the following Docker command to authenticate to the registry (using the `username` and `password` values copied in step 1):
+Create a Google Cloud SQL instance with the following attributes:
```

Comment:
@swampie @tcrespog Any engineering advice on the prescriptive guidance we give for sizing DB and VM instances for Enterprise deployment? I know this depends heavily on the traffic and concurrent pipeline usage the Enterprise instance is expected to handle, but can we hone in on good minimums and a general range to advise customers?

Reply:
> Any engineering advice on the prescriptive guidance we give for sizing DB and VM instances for Enterprise deployment?

I don't know for sure. Perhaps, even if it's deployed on platform different from GCP, those used for cloud.seqera.io might provide a hint (I don't know what are those currently).